### DISCUSSION TITLE
DriftRetrainingClassifier

### DISCUSSION THREAD

**Discussion by kaii55** [online-ml/river#1285] [Q&A] [answered]
I am getting an error while calling the "DriftRetrainingClassifier". 

AttributeError: module 'river.drift' has no attribute 'DriftRetrainingClassifier'

I checked the functions inside the "drift" and there is no "DriftRetrainingClassifier".

print(dir(drift))
['ADWIN', 'DDM', 'EDDM', 'HDDM_A', 'HDDM_W', 'KSWIN', 'PageHinkley', '__all__', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'adwin', 'adwin_c', 'ddm', 'eddm', 'hddm_a', 'hddm_w', 'kswin', 'page_hinkley']

Am I doing anything wrong? 
I will be highly obliged.

**Comment by MaxHalford** [comment, marked as answer]
Hello. What version of River are you using? It might be that your version is not up-to-date with the documentation you're looking at.

**Reply by kaii55** [reply]
Hi Max, 
I was using '0.10.1' version. Then I installed RIVER in a separate virtual environment where the default version came as '0.13.0'. Still it is saying the same error. Also, I used this "pip install git+https://github.com/online-ml/river --upgrade" command to upgrade but it is saying that "ERROR: Could not build wheels for river, which is required to install pyproject.toml-based projects". I am quite confused. A little help is much needed. 
Thanking you.

**Reply by kaii55** [reply]
Hi Max, 
I have now upgraded the version to '0.18.0' and it seems everything is working just fine. 
Thanks you so much for the help. 
Yours sincerely,
Aritra
